appcheck/firebase-appcheck-debug-testing/gradle.properties

```diff
@@ -1,2 +1,2 @@
 version=16.1.1
```

Comment:
we also need to bump version

Reply:
`16.1.1` is the latest release, and there's not any changes warranting a release- unless I'm missing something?

Reply:
`version` is the version used when generating the artifact if you do a gradle build. Since it's not necessarily the exact same code as the released version, it should be `current_live_version + 0.0.1`

Reply:
latestReleasedVersion = what's in maven
version = next version

Reply:
Ah, that's a good point, I didn't think about the fact that Gradle utilizes it. Could cause some issues during development otherwise, especially regarding caching. I'll go ahead and bump the rest as well